docs/email/postfix/email-with-postfix-dovecot-and-mysql/index.md

```diff
@@ -489,6 +489,15 @@ smtps     inet  n       -       -       -       -       smtpd
 
         sudo systemctl restart postfix
 
+    If you get `fatal: 0.0.0.0:smtps: Servname not supported for ai_socktype`
+error after restarting Postfix, add following service entries to
+`/etc/services`:
```

Comment:
I split this paragraph into multiple lines, instead of single line per paragraph